setup.py

```diff
@@ -70,6 +70,7 @@
     "xgboost": ["xgboost"],
     "lightgbm": ["lightgbm"],
     "fastapi": ["uvicorn", "fastapi"],
+    "streamlit": ["uvicorn", "fastapi", "streamlit", "streamlit_pydantic"],
     "sagemaker": ["boto3", "sagemaker"],
```

Comment:
maybe:
```suggestion
    "sagemaker": ["docker", "boto3", "sagemaker"],
```
? @mike0sv 
or you had some idea behind this as well?

Reply:
probably not :)